spec/lib/email_repair/mechanic_spec.rb

```diff
@@ -26,7 +25,7 @@ module EmailRepair
     end
 
     it 'returns nil for strings that have no email' do
-      ['', ' ', 'NOT AN EMAIL', 'b at b dot com'].each do |not_email|
+      ['', ' ', 'NOT AN EMAIL', 'b at b dot com', 'a.@com'].each do |not_email|
```

Comment:
Yeah, but I just wanted to make sure that it was invalid.  (this is the only suite of tests that does it).  Should I add a comment to clarify? or change the name of the spec?